rust/private/rustc.bzl

```diff
@@ -660,6 +660,9 @@ def construct_arguments(
     # Ensure the sysroot is set for the target platform
     env["SYSROOT"] = toolchain.sysroot
 
+    env["RULES_RUST_RENAME_FIRST_PARTY_CRATES"] = str(toolchain._rename_first_party_crates).lower()
```

Comment:
Would it be possible to avoid always adding this to the action?

Reply:
I suppose so - in principle we only need an `Option<String>` in the rust macro. I'll make that change.

In terms of best practices for a build system, is it ok to only set the env var some of the time? I'm imagining things breaking if a user has a `RULES_RUST_THIRD_PARTY_DIR` env var set, but the build setting to rename things wasn't enabled. I guess that's probably unlikely enough that we're ok with that risk?

Reply:
I don't think there's a risk but it would invalidate all targets which I don't think is entirely necessary. If this becomes core to how all targets are built then I think it'd be fine to always include it but for now my understanding is that this is opt-in functionality and I think it's better to not unnecessarily include related data into all builds even when unused.

Reply:
Ah, of course! That didn't occur to me, but makes total sense.